docs/vagrant.md

```diff
@@ -23,51 +30,15 @@ To ssh in the cluster, run following:
 vagrant ssh
 ```
 
-The password for vagrant user is **vagrant'**
-
-## Setting Up YARN/Hadoop ##
-
-To setup YARN/Hadoop inside VM, run following YARN setup shell files:
-
-1 Run the first YARN setup shell command from the vagrant directory to create a user hduser in group hadoop. Be sure to remember the password that you provide for this user.
-```
-cd /vagrant
```

Comment:
I'll add the headers, and switch to Java 7.

On Wednesday, November 11, 2015, Santosh Marella <notifications@github.com>
wrote:

> In docs/vagrant.md
> <https://github.com/apache/incubator-myriad/pull/43#discussion_r44590149>:
>
> > @@ -23,51 +30,15 @@ To ssh in the cluster, run following:
> >  vagrant ssh
> >  ```
> >
> > -The password for vagrant user is **vagrant'**
> > -
> > -## Setting Up YARN/Hadoop ##
> > -
> > -To setup YARN/Hadoop inside VM, run following YARN setup shell files:
> > -
> > -1 Run the first YARN setup shell command from the vagrant directory to create a user hduser in group hadoop. Be sure to remember the password that you provide for this user.
> > -```
> > -cd /vagrant
>
> Looks like the vagrant setup is a lot simpler now. Would you mind updating
> the wiki page as well?
>
> https://cwiki.apache.org/confluence/display/MYRIAD/Installing+using+Vagrant
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/43/files#r44590149>.
>